pinot-tools/src/main/java/com/linkedin/pinot/tools/perf/QueryRunner.java

```diff
@@ -673,6 +692,44 @@ public void run() {
     }
   }
 
+  @ThreadSafe
+  private static class Statistics {
```

Comment:
Thought about that, but that will make adding values difficult (I don't want to store a map inside because adding value could be a very frequent operation).